docs/integrations/gitpod.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Using Steampipe in Gitpod
+sidebar_label: Gitpod
+---
+# Using Steampipe in Gitpod
+
+[Gitpod](https://www.gitpod.io/) is an open source platform provisioning ready-to-code developer environments that integrates with GitHub. Here we integrate a Github project with Gitpod to install Steampipe, then install a plugin and run a query.
+
+## Installing Steampipe in Gitpod
+
+To run scripts, first connect your GitHub repository to Gitpod workspace and create a `.gitpod.yml` file that contains the definitions. Here's an example that installs Steampipe.
+
+```yaml
+tasks:
+  - name: Install Steampipe with RSS Plugin
+    init: |
+      sudo /bin/sh -c "$(curl -fsSL https://raw.githubusercontent.com/turbot/steampipe/main/install.sh)"
+      steampipe -v
+
+```
+
+## Running Steampipe in Gitpod
+
+In order to run Steampipe commands, we will first install the [RSS](https://hub.steampipe.io/plugins/turbot/rss) plugin.
+
+```yaml
+tasks:
+  - name: Install Steampipe with RSS Plugin
+    init: |
+      sudo /bin/sh -c "$(curl -fsSL https://raw.githubusercontent.com/turbot/steampipe/main/install.sh)"
+      steampipe -v
+      steampipe plugin install steampipe
+      steampipe plugin install rss
+
+ports:
+ # Steampipe/ PostgreSQL
+  - port: 9193
+
+```
+
+<div style={{"marginBottom":"2em","borderWidth":"thin", "borderStyle":"solid", "borderColor":"lightgray", "padding":"20px", "width":"50%"}}>
```

Comment:
images are missing in the preview here: https://steampipe-m1k1m54w1-turbot.vercel.app/docs/integrations/gitpod

(I found the preview by way of https://cloud.steampipe.io/org/acme/workspace/jon/dashboard/steampipe_stats.dashboard.Vercel)

Reply:
The images were in a different directory. I just changed it. It should show up in the preview now.